components/alerts/src/css/simple-alert.css

```diff
@@ -0,0 +1,227 @@
+}
+.success-button {
+  display: block;
+  padding: 1rem;
+  margin: 2rem;
+  background-color: var(--font-color);
+  color : var(--background-color);
+  font-size: 1.5rem;
+  cursor: pointer;
+}
+.alert-container {
+  height: 25rem;
+  position: relative;
+  width: 100%;
+  margin: 0;
+}
+
+.simple-alert .preview{
+  margin: 0;
+  display: none;
+  padding: 1rem;
+  font-size: 1.5rem;
+}
+
+.dismiss-alert .preview{
+  margin: 0;
+  display: none;
+  padding: 1rem;
+  font-size: 1.5rem;
+}
+
+.warning-alert {
+  background: #ffdb9b;
+  position: absolute;
+  padding: 2rem 4rem;
+  top: 0;
+  right: 0;
+  border-radius: 1rem;
+  border-left: 8px solid #ffa502;
+  pointer-events: none;
+  visibility: hidden;
+}
+
+.warning-alert .fa-exclamation-circle {
+  position: absolute;
+  left: 20px;
+  top: 50%;
+  transform: translateY(-50%);
+  color: #ce8550;
+}
+
+.warning-alert .warning-msg {
+  padding: 0 20px;
+  font-size: 1.8rem;
+  color: #ce8550;
+}
+
+.alert-container .showAlert {
+  visibility: visible;
+  pointer-events: auto;
+}
+
+.alert-container .show {
+  animation: show_slides 1s ease forwards;
+}
+
+@keyframes show_slides {
+  0% {
+    transform: translateX(100%);
+  }
+  40% {
+    transform: translateX(-10%);
+  }
+  80% {
+    transform: translateX(0%);
+  }
+  100% {
+    transform: translateX(-10px);
+  }
+}
+
+/* Success Alert */
+
+.success-alert {
+  background: #d9ff9b;
+  position: absolute;
+  padding: 2rem 4rem;
+  top: 6.5rem;
+  right: 0;
+  border-radius: 1rem;
+  border-left: 8px solid #89ff02;
+  pointer-events: none;
+  visibility: hidden;
+}
+
+.success-alert .fa-check-circle {
+  position: absolute;
+  left: 20px;
+  top: 50%;
+  transform: translateY(-50%);
+  color: #97ce50;
+}
+
+.success-alert .success-msg {
+  padding: 0 20px;
+  font-size: 1.8rem;
+  color: #97c350;
+}
+
+/* Info Alert */
+
+.info-alert {
+  background: #9bc6ff;
+  position: absolute;
+  padding: 2rem 4rem;
+  top: 13rem;
+  right: 0;
+  border-radius: 1rem;
+  border-left: 8px solid #3902ff;
+  pointer-events: none;
+  visibility: hidden;
+}
+
+.info-alert .fa-info {
+  position: absolute;
+  left: 20px;
+  top: 50%;
+  transform: translateY(-50%);
+  color: #507cce;
+}
+
+.info-alert .info-msg {
+  padding: 0 20px;
+  font-size: 1.8rem;
+  color: #507cce;
+}
+
+/* Danger Alert */
+.danger-alert {
+  background: #ff9b9b;
+  position: absolute;
+  padding: 2rem 4rem;
+  top: 19.5rem;
+  right: 0;
+  border-radius: 1rem;
+  border-left: 8px solid #ff0a02;
+  pointer-events: none;
+  visibility: hidden;
+}
+
+.danger-alert .fa-exclamation-triangle {
+  position: absolute;
+  left: 20px;
+  top: 50%;
+  transform: translateY(-50%);
+  color: #ce5050;
+}
+
+.danger-alert .danger-msg {
+  padding: 0 20px;
+  font-size: 1.8rem;
+  color: #ce5050;
+}
+
+/* Dismiss Button */
+.warning-alert .fa-times-circle {
+  position: absolute;
+  right: 20px;
+  top: 50%;
+  transform: translateY(-50%);
+  color: #ce8550;
+  cursor: pointer;
+}
```

Comment:
Try to use variables for colors. Rest all good to go.

Reply:
There are like 3 colors for each alert, so there are like 12 colors that I am not using anywhere other than the alert.
Also I will be deleting the other file after entire completion of the CL, I am keeping it so I can have the folder structure and and all the CSS in one folder

Reply:
Ok cool I am approving now. Make sure you structure it properly after getting done.